native/src/routes/news/containers/NewsContainer.js

```diff
@@ -39,15 +39,16 @@ type ContainerPropsType =
       newsId: ?string,
       language: string,
       cityModel: CityModel,
-      selectedNewsType: NewsType
+      selectedNewsType: NewsType,
+      page: null
     |}
   | {|
       ...NavigationPropsType,
       ...DispatchPropsType,
       status: 'ready',
       news: NewsModelsType,
       hasMoreNews: ?boolean,
-      page: ?number,
+      page: number | null,
```

Comment:
why this change?

Reply:
This is to avoid sketchy 0 check, i.e. evaluation to null if page is 0.